src/ArchivItem_Map_Header.cpp

```diff
@@ -280,3 +280,47 @@ void libsiedler2::ArchivItem_Map_Header::setAuthor(const std::string& author)
 {
     this->author_ = author;
 }
+
+/**
+ * Sets the x-coordinate of the headquarter for player n.
+ *
+ * @param n player number (0-6)
+ * @param x x-coordinate of the headquarter
+ */
+void libsiedler2::ArchivItem_Map_Header::setPlayerHQx(const int n, unsigned short x)

```

Comment:
why not create one

```
setPlayerHQ(const unsigned int player, const unsigned short x, const unsigned short y)
```

instead of setPlayerHQx and setPlayerHQy